core/tests/webdriverio_desktop/explorationFeedbackTab.js

```diff
@@ -216,13 +216,15 @@ describe('ExplorationFeedback', function() {
     await explorationEditorPage.navigateToFeedbackTab();
     await explorationEditorFeedbackTab.sendResponseToLatestFeedback(
       feedbackResponse);
+    await explorationEditorFeedbackTab.expectNumberOfFeedbackMessagesToBe(2);
```

Comment:
Why this check now?

Reply:
All the feedback elements take some time to appear, so we need to wait till all the feedback are visible and then proceed further else we are unable to get the feedback elements in further steps.